src/Components/PairingForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { findGame } from '../redux/actions/boardGame';
+import { setLiquorType, setRandomDrink } from '../redux/actions/liquorSearch'
+
+const PairingForm = () => {
+
+    const dispatch = useDispatch()
+
+    const games = useSelector(state => state.boardGame.allGamesData.games)
+    const type = useSelector(state => state.liquorSearch.liquorSearchWord)
+
+    const gameNames = games.map((game) => {
+        return (
+            <option key={game.id} value={game.name} />
+        )
+    })
+
+    const findRandomDrink = (event) => {
```

Comment:
For naming, what if we called this 'getRandomDrink' instead of 'findRandomDrink'? What do y'all think? It really doesn't matter, but seems a little more accurate to me.